src/components/SideBar/SideBar.tsx

```diff
@@ -20,7 +20,7 @@ const SideBar = () => {
   return (
     <Box sx={styles.rightContent}>
       <Box sx={styles.searchBarContainer}>
-        <SearchBar placeholder="Search Chirp" />
+        <SearchBarDropdown placeholder="Search Chirp" />
```

Comment:
Nit, can we make the placeholder "Search users"? Same with the `ExpandedPostSidebar`